When other bits of the source got fixed to use a different include path
for efivars, the test directory did not.

--- src/test/tester.c
+++ src/test/tester.c
@@ -26,7 +26,7 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
-#include <efivar.h>
+#include <efivar/efivar.h>
 
 #define TEST_GUID EFI_GUID(0x84be9c3e,0x8a32,0x42c0,0x891c,0x4c,0xd3,0xb0,0x72,0xbe,0xcc)
 

